exporter/containerimage/writer.go

```diff
@@ -65,6 +65,9 @@ func (ic *ImageWriter) Commit(ctx context.Context, inp *exporter.Source, session
 	}
 
 	multiPlatform := len(inp.Refs) > 0
+	if len(inp.Attestations) > 0 {
```

Comment:
if the `len(inp.Refs) > 0` clause above is wrong then it should be removed. If it is not wrong then this patch does not maintain that behavior.